gem/migrations/0018_add_extra_ga_account.py

```diff
@@ -0,0 +1,30 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.9.13 on 2017-08-10 13:01
+from __future__ import unicode_literals
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('gem', '0017_show_join_banner_setting'),
+    ]
+
+    operations = [
+        migrations.AddField(
+            model_name='gemsettings',
+            name='extra_ga_account',
```

Comment:
I think you'll need to be explicit that this is for BBM. Especially since the business logic under the hood looks at this field and the subdomain one. 

I would recommend "BBM GA Account" and "BBM Subdomain"